packages/docs/src/components/ButtonVariationsTable.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { ReactElement } from 'react';
+import {
+  Badge,
+  Button,
+  Table,
+  TableCaption,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+} from '@cmsgov/design-system';
+
+interface VariationButtonUsage {
+  main?: {
+    onLight?: boolean;
+    onDark?: boolean;
+  };
+  alternate?: {
+    onLight?: boolean;
+    onDark?: boolean;
+  };
+}
+
+interface ThemeButtonUsage {
+  outline?: VariationButtonUsage;
+  solid?: VariationButtonUsage;
+  ghost?: VariationButtonUsage;
+}
+
+const themeButtonUsage: { [theme: string]: ThemeButtonUsage } = {
+  healthcare: {
+    outline: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    solid: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    ghost: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+  },
+  medicare: {
+    outline: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    ghost: {
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+  },
+};
+
+export interface ButtonVariationsTableProps {
+  /**
+   * Name of currently selected theme
+   */
+  theme: string;
+}
+
+const ButtonVariationsTable = ({ theme }: ButtonVariationsTableProps) => {
+  const uses = themeButtonUsage[theme] ?? {};
+  const unused = <Badge variation="alert">Unused</Badge>;
+
+  function b(buttonInstance: ReactElement, isUsed?: boolean) {
+    return isUsed === false ? unused : buttonInstance;
+  }
+
+  return (
+    <>
+      <Table stackable stackableBreakpoint="md" className="c-button-variations-table">
+        <TableCaption>Table of button variations on light background</TableCaption>
+        <TableHead>
+          <TableRow>
+            <TableCell scope="col">Variation</TableCell>
+            <TableCell scope="col" className="ds-u-font-weight--normal">
+              Main <small>(Default)</small>
+            </TableCell>
+            <TableCell scope="col" className="ds-u-font-weight--normal">
+              Alternate
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
```

Comment:
Consider adding the prop `stackedTitle` to all table body `<TableCell>` components. Currently on mobile, the table stacks and looks fine, but it feels like a bit of context is missing since the column titles are not automatically added
<img width="289" alt="Screen Shot 2022-07-27 at 10 21 27 AM" src="https://user-images.githubusercontent.com/33579665/181299322-4d8913d3-493a-4d42-b9d1-62c1a5fe81fe.png">